app/controllers/admin/username_blocks_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class Admin::UsernameBlocksController < Admin::BaseController
+  before_action :set_username_block, only: [:edit, :update]
+
+  def index
+    authorize :username_block, :index?
+    @username_blocks = UsernameBlock.order(normalized_username: :asc).page(params[:page])
```

Comment:
I think it would be less confusing for moderators if it were sorted based on `username` (or `lower(username)`) not `normalized_username` which has substitutions.